honeyscanner/active_attacks/honeypot_port_scanner/honeypot_port_scanner.py

```diff
@@ -48,17 +50,22 @@ def scan_honeypot(self):
             else:
                 self.report['os'] = []
 
-            ports: Report = {}
+            report_ports: Report = {}
+            attack_ports: AttackPorts = {}
             for port_info in host_info['ports']:
                 port = port_info['portid']
-                ports[port] = {
+                report_ports[port] = {
                     'name': port_info['service']['name'] if 'name' in port_info['service'] else '',
                     'product': port_info['service']['product'] if 'product' in port_info['service'] else '',
                     'version': port_info['service']['version'] if 'version' in port_info['service'] else '',
                     'cpe': port_info['service']['cpe'] if 'cpe' in port_info['service'] else [],
                     'script': port_info['service']['script'] if 'script' in port_info['service'] else []
                 }
-            self.report['ports'] = ports
+                attack_ports[port] = {
```

Comment:
this needs to be adjusted as now we will run a port scan only, not scanning with scripts (-sC) and versions (-sV) which are parts of the (-A) flag. Adjustments need to be made.